### PR TITLE
feat: La Molina

### PR DESCRIPTION
**Pull request by FezVrasta**
I must be blonde but I keep having problems with these selectors... Why isn't this one working?

### CODE REVIEW

File: lib/resorts/lamolina/index.js [hunk 1]
```diff
@@ -1,22 +1,7 @@
-const toTitleCase = require("to-title-case");
-
 module.exports = {
-  selector: '[name=remontadores] + .tabla table tbody tr',
+  selector: '[name=remontadores] + .tabla tbody tr',
```
Comment (FezVrasta) [inline lib/resorts/lamolina/index.js:2]: Why did you have to remove `table` from here? The markup is `.tabla > table`

**Comment by FezVrasta** [comment]
Could you merge this when you have time please?

**Comment by pirxpilot** [comment]
Merged, Thanks. In future please squash the history and force push to PR branch.

**Comment by FezVrasta** [comment]
You can squash through the github UI, isn't that simpler?

**Pull request closed without merging**